app/models/claims/school.rb

```diff
@@ -48,4 +48,5 @@ class Claims::School < School
   default_scope { claims }
 
   has_many :users, -> { claims }, through: :memberships
+  has_many :claim_applications, class_name: "Claim"
```

Comment:
Would this be more succinct if modelled as "claims"?

```suggestion
  has_many :claims
```

Reply:
I did use has many claims but it conflicts with the `claims` boolean attribute on the school record.

Reply:
D'oh! 🤦🏻 Makes me wish we'd named the boolean attribute `claims_enabled`, because it feels more natural for this association to be called `claims`.